### DISCUSSION TITLE
Uninstallation

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8577] [Q&A] [answered]
Hello,

Could you please help me with uninstallation script / steps for MAC Machine ?

Also, I am not seeing any KeePassXC location at below registry location:

![image](https://user-images.githubusercontent.com/108385220/195834079-7126fdb9-519a-472b-87f3-0eef446f950b.png)

Could you please help me with all the registry entries for KeePassXC in Windows Machine ?

Thanks

**Comment by phoerious** [comment, marked as answer]
Unless you enabled the browser extension, we don't set any registry entries. Just use the provided uninstaller.

**Reply by savidudu** [reply]
If possible could you please help with the macOS uninstall process ?
I am bit lost and finding difficult to remove it from machine m.
Is there any script or steps for macOS uninstallation.
Thanks

**Reply by phoerious** [reply]
Just delete the app from your /Applications folder.
